openedx_tagging/core/tagging/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Tagging API URLs.
+"""
+
+from django.urls import path, include
+
+from .rest_api import urls
+
```

Comment:
We need an `app_name` here in order to use [the namespace I recommended below](https://github.com/open-craft/openedx-learning/pull/4/files#diff-736f67c058debd0bdd57a6bf62f4c9caf366ca6223e987fd3996a989b0d2fe7dR12):

```suggestion

app_name = "oel_tagging"
```